setup.py

```diff
@@ -64,6 +64,25 @@ def finalize_options(self):
         install.finalize_options(self)
 
 
+cmdClass = {'install': CheckCairoXcb}
+
+try:
+    from sphinx.setup_command import BuildDoc
+    import subprocess
+
+    class BuildDocWrapper(BuildDoc):
+        def run(self):
+            subprocess.call(['scripts/ffibuild'])
+            subprocess.call(['make', '-C', 'docs', 'genkeyimg'])
```

Comment:
Why not doing everything so that we don't need the `-C docs`?

Reply:
We hold `docs` separately. All logics here. I think it should leave here as it is

Reply:
Not sure to understand but it's no big deal :)